src/crimson/osd/scheduler/scheduler.cc

```diff
@@ -19,27 +19,10 @@
 #include "crimson/osd/scheduler/scheduler.h"
 #include "crimson/osd/scheduler/mclock_scheduler.h"
 #include "common/WeightedPriorityQueue.h"
+#include "common/mclock_common.h"
 
 namespace crimson::osd::scheduler {
 
-std::ostream &operator<<(std::ostream &lhs, const scheduler_class_t &c)
```

Comment:
Why was this removed?

Reply:
It is not required because it is a part of of the common code.